README.md

```diff
@@ -38,7 +38,7 @@ Each transcript is a markdown file, which requires to include the `title` and `t
 ---
 title: Carl Dong - Reproducible Builds (2020-11-30)
 transcript_by: Bryan Bishop
-speaker: Carl Dong
```

Comment:
There are 45 transcripts formatted like this. I don't think it makes sense to change this standard for tweeting purposes.